libgraph/src/BuildGraph.cpp

```diff
@@ -1895,7 +1896,11 @@ katana::WritePropertyGraph(
         "edge prop: ({}) {}", field->type()->ToString(), field->name());
   }
 
-  auto result = prop_graph.Write(dir, "libkatana_graph");
+  // Since this function is only used by tools under
+  // external/katana/tools/graph-convert/, but not any other graph
```

Comment:
I agree with this suggestion.

Reply:
@ddn0 @witchel 
The function `WritePropertyGraph` is used only by tools under `tools/graph-convert`. But there are lots of other functions or classes in `BuildGraph` used by other parts in the repo, like `libgraph`. There are also lots of structs and functions not used by any other code in the repo and probably need to be removed.

Since the above comment in code is out-of-dated after switching from using the boolean to `TxnContext`, I'll just remove the comment and leave the code in `BuildGraph.cpp`? Or do you still want me to move the function to somewhere like `tools/graph-convert/Transform.h`?

Reply:
Let's focus on landing your PR.  We need to keep  `PropertyGraph::Write`,  so I agree you can just change to comment to say this Write is committing directly to storage because this is a non-transactional tool.

I guess my question is about `PropertyGraph::Write` and `PropertyGraph::Commit`.  Do these require a TxnContext?

Reply:
These require `TxnContext` mainly for caching the `RDGManifest`. I'm going to write it so that when `Commit` is called, it does not writeback `RDGManifest`, but writes the manifest to `TxnContext` instead. (Before the current changes, it was for checking on `commit_manifest_`)